core/src/main/scala/com/novocode/ornate/config/Global.scala

```diff
@@ -99,7 +99,7 @@ class Global(startDir: File, confFile: Option[File], overrides: Config = ConfigF
   def findSources: Vector[(File, String, URI)] = {
     val dir = userConfig.sourceDir
     val suffix = ".md"
-    dir.collectChildren(f => f.isRegularFile && f.name.endsWith(suffix)).map { f =>
+    dir.collectChildren(f => f.isRegularFile && f.name.endsWith(suffix)).toSeq.map { f =>
```

Comment:
Transformation to strict collection, because otherwise was getting IllegalStateException in slick for only 2.13.1. No exception for 2.11.12 and 2.12.9. Seems to be this issue: pathikrit/better-files#270.